core/builtins/src/kotlin/Range.kt

```diff
@@ -27,6 +27,8 @@ public interface ClosedRange<T: Comparable<T>> {
 
     /**
      * Checks whether the range is empty.
+     *
+     * The closed range is empty if its start value is greater than or equal to the end value.
```

Comment:
"equal to" is not correct, see the condition below.

Reply:
Returned the wording as for the usual `Range`